### PR TITLE
Fixing Dark Mode

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
In this PR:

- Fixing the colour of the view for typing status. The bar looks a bit lighter now.

| Before | After | 
| --- | --- | 
| Screenshot_20210218-120109 | Screenshot_20210218-120019 | 
| Screenshot_20210218-120059 | Screenshot_20210218-120005 |

### CODE REVIEW

File: samplekotlin/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -20,9 +20,17 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-        <activity android:name=".ChannelActivity" />
-        <activity android:name=".ChannelActivity2" />
-        <activity android:name=".ChannelActivity3" />
-        <activity android:name=".ChannelActivity4" />
+        <activity android:name=".ChannelActivity"
+            android:theme="@style/AppTheme.NoActionBar"
+            />
```
Comment (zsmb13) [inline samplekotlin/src/main/AndroidManifest.xml]: This doesn't look spectacular in the tutorial, but it's not worth adding extra steps for

File: samplekotlin/src/main/res/layout/activity_channel_3.xml [hunk 2]
```diff
@@ -16,8 +16,9 @@
         android:id="@+id/typingHeaderView"
         android:layout_width="0dp"
         android:layout_height="31dp"
-        android:background="#CCCCCC"
+        android:background="@color/light_gray"
```
Comment (zsmb13) [inline samplekotlin/src/main/res/layout/activity_channel_3.xml]: This small improvement needs too many file modifications, it's not worth complicating the tutorial with 3 file edits to get some colours right. At most we should find one better looking colour for both dark and light mode, and if there isn't one, then we can leave the original.

**Review by zsmb13 — CHANGES_REQUESTED** [review]

File: samplekotlin/src/main/AndroidManifest.xml (hunk 1 above)
Comment (leandroBorgesFerreira) [reply samplekotlin/src/main/AndroidManifest.xml]: What about changing `AppTheme` from `Theme.AppCompat.Light.DarkActionBar` to `Theme.MaterialComponents.DayNight.NoActionBar` I don't see the need of a `ActionBar`, what do you think?

File: samplekotlin/src/main/res/layout/activity_channel_3.xml (hunk 2 above)
Comment (leandroBorgesFerreira) [reply samplekotlin/src/main/res/layout/activity_channel_3.xml]: 👍

**Review by zsmb13 — APPROVED** [review]

**Pull request merged**